application/src/main/kotlin/com/henriquebarucco/ssbc/sensor/get/DefaultGetSensorUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package com.henriquebarucco.ssbc.sensor.get
+
+import com.henriquebarucco.ssbc.sensor.SensorGateway
+import com.henriquebarucco.ssbc.sensor.dto.GetSensorDto
+import com.henriquebarucco.ssbc.sensor.get.dto.GetSensorOutput
+import com.henriquebarucco.ssbc.sensor.get.dto.GetSensorQuery
+
+class DefaultGetSensorUseCase(
+    private val sensorGateway: SensorGateway,
+) : GetSensorUseCase() {
+    override fun execute(input: GetSensorQuery): GetSensorOutput? {
```

Comment:
Podemos fazer o GetSensorOutput sempre retornar pois é output de usecase

```suggestion
    override fun execute(input: GetSensorQuery): GetSensorOutput {
```